platform/emdrv/nvm3/src/ccp_flash_intf.c

```diff
@@ -0,0 +1,349 @@
+/*******************************************************************************
+ * @file  Flash_Intf.c
+ * @brief
+ *******************************************************************************
+ * # License
+ * <b>Copyright 2020 Silicon Laboratories Inc. www.silabs.com</b>
+ *******************************************************************************
+ *
+ * The licensor of this software is Silicon Laboratories Inc. Your use of this
+ * software is governed by the terms of Silicon Labs Master Software License
+ * Agreement (MSLA) available at
+ * www.silabs.com/about-us/legal/master-software-license-agreement. This
+ * software is distributed to you in Source Code format and is governed by the
+ * sections of the MSLA applicable to Source Code.
+ *
+ ******************************************************************************/
+#include "ccp_flash_intf.h"
+#include "RS1xxxx_9117.h"
+#include "rsi_board.h"
+#include "rsi_chip.h"
+#include "system_RS1xxxx.h"
+
+uint32_t RSI_FLASH_UnInitialize(void) {
+  /* This function not supported so always returns zero */
+  return 0;
+}
+
+void qspi_gpio_revert_m4(void) {
+  uint32_t gpio_pin_no = 0;
+  uint32_t gpio_mode = VAL_15;
```

Comment:
Not a fan of those VAL_15, VAL_12 ,VAL_11, etc. defines. Those name do not add much to the readability. Not a blocker but something to reflect.